### ISSUE TITLE
node-gyp configure -> fail on Windows 7 x64

### ISSUE THREAD

**Issue by fperreault** [nodejs/node-gyp#155]
I don't know why.
Can you help me.

```
C:\Users\fperreault\Desktop\hid>node-gyp configure
gyp info it worked if it ends with ok
gyp info using node-gyp@0.7.1
gyp info using node@0.8.12 | win32 | x64
gyp ERR! configure error
gyp ERR! stack Error: Command failed:   File "<string>", line 1
gyp ERR! stack     import platform; print platform.python_version();
gyp ERR! stack                                   ^
gyp ERR! stack SyntaxError: invalid syntax
gyp ERR! stack
gyp ERR! stack     at ChildProcess.exithandler (child_process.js:540:15)
gyp ERR! stack     at ChildProcess.EventEmitter.emit (events.js:96:17)
gyp ERR! stack     at maybeClose (child_process.js:638:16)
gyp ERR! stack     at Process._handle.onexit (child_process.js:680:5)
gyp ERR! System Windows_NT 6.1.7601
gyp ERR! command "node" "C:\\Users\\fperreault\\AppData\\Roaming\\npm\\node_modu
les\\node-gyp\\bin\\node-gyp.js" "configure"
gyp ERR! cwd C:\Users\fperreault\Desktop\hid
gyp ERR! node -v v0.8.12
gyp ERR! node-gyp -v v0.7.1
gyp ERR! not ok
```

**Comment by WarFox** [comment]
Python 2.7, not Python 2.6.

I can confirm `node-gyp rebuild` did not work on `npm install java` (https://github.com/joeferner/node-java) with python `2.6.6` for 'node 4.1.1`

**Comment by avnersch** [comment]
`node-gyp rebuild` does not work for me too on `node-java`.
I am using Python 2.7.6.

**Comment by jmusser73** [comment]
```node-gyp rebuild``` is not working for ```node-java```.
I am using Python 2.7.8

**Comment by WarFox** [comment]
have you checked you GCC version? It needs to be above 4.8, is what I remember.

**Comment by aravindtrue** [comment]
@WarFox Is there a fix for this issue? I'm using Python 2.7.15 and facing the same issue.
